R/def_dislnorm.R

```diff
@@ -208,16 +211,18 @@ setMethod("dist_rand",
 dislnorm$methods(
   mle = function(set = TRUE, initialise=NULL) {
     n = internal[["n"]]
```

Comment:
We don't use `n` - so remove this line

Reply:
just to be clear, that line was there before me.

Reply:
Not sure why I had it in. Just delete. Thanks